sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -44,12 +47,28 @@ fun HomeScreen(
     )
 }
 
+@OptIn(ExperimentalPermissionsApi::class)
 @Composable
 private fun HomeScreenContent(
     uiModels: List<UiModel>,
     showLoading: IsLoading,
     onItemClick: (UiModel) -> Unit
 ) {
+    val cameraPermissionState = rememberPermissionState(CAMERA)
+    if (cameraPermissionState.status.isGranted) {
+        Timber.d("${cameraPermissionState.permission} granted")
+    } else {
+        if (cameraPermissionState.status.shouldShowRationale) {
+            Timber.d("${cameraPermissionState.permission} needs rationale")
+        } else {
+            Timber.d("Request cancelled, missing permissions in manifest or denied permanently")
+        }
+
+        LaunchedEffect(Unit) {
+            cameraPermissionState.launchPermissionRequest()
+        }
+    }
```

Comment:
As the sample, could we show on Toast instead of Timber? 🤔

Reply:
@hoangnguyen92dn I think Timber logging is good enough. Adding Toast will require us to take care of more things from Toast implementation, it could be an easy step but out of scope 🙏

Reply:
@luongvo Yeah. My idea is that simply launch the application then play around to see the result rather than take a look at the log. Anw, it's not a big deal. We could skip it for now.

Reply:
@hoangnguyen92dn Fixed in https://github.com/nimblehq/android-templates/pull/375/commits/decb5e2d24caf91d6a9ac742d4ea717c5b24f17c